server/@types/crm7/index.d.ts

```diff
@@ -14,6 +14,24 @@ type NumberRateAndCost = {
 }
 
 export interface Crm7Response extends CrmResponse {
+  mergedScheduleCostsData: {
```

Comment:
I think we should remove `mergedScheduleCostsData` from here as Crm7Response type should represent the data returned from the API.